### PR TITLE
Copy over changes from mwaterw-add-field-functionality

### PR DESCRIPTION
**Pull request by waterwo2**
Added func to TextFields, put some TODOs for other stories
Added dropdown func for Color select

Jira Story: https://tim-kaufenberg.atlassian.net/browse/ARC-87

Created this branch to more easily merge in changes from mwaterw-add-field-functionality

### CODE REVIEW

File: ui/src/App.tsx [hunk 1]
```diff
@@ -157,9 +221,10 @@ const App = () => {
           </div>
           <Divider />
           <Typography style={{margin:"10px"}}>1. Enter your dimensions</Typography>
-          <TextField style={{margin:"10px"}} id="outlined-basic" label="Height" variant="outlined" />
-          <TextField style={{margin:"10px"}} id="outlined-basic" label="Weight" variant="outlined" />
-          <TextField style={{margin:"10px"}} id="outlined-basic" label="Armspan" variant="outlined" />
+          <TextField style={{margin:"10px"}} id="outlined-basic" label="Height (ft)" variant="outlined" type="number" onChange={(e)=>{handleOnChange(e, "hf")}}/>
+          <TextField style={{margin:"10px"}} id="outlined-basic" label="Height (in)" variant="outlined" type="number" onChange={(e)=>{handleOnChange(e, "hi")}}/>
+          <TextField style={{margin:"10px"}} id="outlined-basic" label="Weight (lbs)" variant="outlined" type="number" onChange={(e)=>{handleOnChange(e, "w")}}/>
+          <TextField style={{margin:"10px"}} id="outlined-basic" label="Armspan (ft)" variant="outlined" type="number" onChange={(e)=>{handleOnChange(e, "a")}}/>
```
Comment (skitscha) [inline ui/src/App.tsx]: Can we change the ids on these text fields to something a little more applicable in each case?

File: ui/src/App.tsx [hunk 2]
```diff
@@ -169,8 +234,33 @@ const App = () => {
...
+              <MenuItem value={"purple"}>Purple</MenuItem>
+              <MenuItem value={"white"}>White</MenuItem>
+              <MenuItem value={"orange"}>Orange</MenuItem>
+              <MenuItem value={"yellow"}>Yellow</MenuItem>
+              <MenuItem value={"brown"}>Brown</MenuItem>
+              <MenuItem value={"grey"}>Grey</MenuItem>
+            </Select>
+          </FormControl>
+
+          <Button style={{margin:"10px"}} variant="contained" color="secondary" onClick={(e)=>{handleSubmit(e)}}> Submit </Button>
```
Comment (skitscha) [inline ui/src/App.tsx]: Can we change the id on this drop down to something a little more applicable?

**Review by skitscha — APPROVED** [review]

File: ui/src/App.tsx [hunk 3]
```diff
@@ -76,23 +81,42 @@ const useStyles = makeStyles((theme) => ({
...
+    const [armspan, setArmspan] = React.useState(0);
+
+    //Color
+    const [color, setColor] = React.useState('');
+
+    //Drawer Open
     const handleDrawerOpen = () => {
         setOpen(true);
-        console.log(true);
+        // console.log(true);
```
Comment (mattshoobs) [inline ui/src/App.tsx]: Remove commented out code

File: ui/src/App.tsx [hunk 4]
```diff
@@ -76,23 +81,42 @@ const useStyles = makeStyles((theme) => ({
...
     };
 
+    //Drawer Close
     const handleDrawerClose = () => {
         setOpen(false);
-        console.log(false);
+        // console.log(false);
```
Comment (mattshoobs) [inline ui/src/App.tsx]: Remove commented out code

File: ui/src/App.tsx [hunk 5]
```diff
@@ -108,6 +132,46 @@ const App = () => {
         })
     }
 
+    //Updating number text fields
+    const handleOnChange = (event: { target: { value: any; }; }, label: string) => {
+      // console.log(event.target.value);
+      // console.log(label);
```
Comment (mattshoobs) [inline ui/src/App.tsx]: Remove commented out code

File: ui/src/App.tsx [hunk 6]
```diff
@@ -108,6 +132,46 @@ const App = () => {
...
+        setHeightIn(event.target.value);
+      }
+      else if(label == "w"){
+        setWeight(event.target.value);
+      }
+      else if(label == "a"){
+        setArmspan(event.target.value);
+      }
+
+      // console.log("hf:" + heightFt + " hi:" + heightIn + " w:" + weight + " a:" + armspan);
```
Comment (mattshoobs) [inline ui/src/App.tsx]: Remove commented out code

File: ui/src/App.tsx [hunk 7]
```diff
@@ -108,6 +132,46 @@ const App = () => {
...
+    };
+
+    //Submit Button Functionality
+    const handleSubmit = (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+      //TODO: send data somewhere and close nav?
+      //TODO: add image check here
```
Comment (mattshoobs) [inline ui/src/App.tsx]: If already implemented remove comments

**Review by skitscha — APPROVED** [review]

**Review by mattshoobs — APPROVED** [review]

**Pull request merged**